.vsts-dotnet-ci.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 - job: CoreBootstrappedOnLinux
   displayName: "Linux Core"
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```

Comment:
Should we peg to `20.04` instead of `latest`? more manual updates later but more explicit for servicing branches.

Reply:
I vote on latest (less steps for the team)

> but more explicit for servicing branches.

has this ever been relevant for linux?

Reply:
Not that I can think of; it's more relevant in SDK/installer land.